src/management-system-v2/lib/helpers.ts

```diff
@@ -93,7 +97,8 @@ async function versionUserTasks(
       let fileName = `${htmlMapping[userTaskId].fileName}-${newVersion}`;
 
       // get the html of the user task in the based on version (if there is one and it is locally known)
-      const basedOnBPMN = await getLocalVersionBpmn(processInfo, versionBasedOn);
+      const basedOnBPMN =
```

Comment:
Wouldn't this become 0 instead of retrieving the (theoretically valid but practically not expected) version 0 of the process when versionBasedOn is 0